<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt5Class</class>
 <widget class="QMainWindow" name="tt5Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tt5</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frameMap">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="mpActionZoomIn"/>
    <addaction name="mpActionZoomOut"/>
    <addaction name="mpActionPan"/>
    <addaction name="mpActionAddLayer"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mpActionZoomIn">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="mpActionZoomOut">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="mpActionPan">
   <property name="text">
    <string>平移</string>
   </property>
  </action>
  <action name="mpActionAddLayer">
   <property name="text">
    <string>加载图层</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tt5.qrc"/>
 </resources>
 <connections/>
</ui>
